File: app/src/main/java/com/ddu/db/entity/StudyContent.java
package com.ddu.db.entity;

import io.objectbox.annotation.Entity;
import io.objectbox.annotation.Id;

/**
 * Created by yzbzz on 2018/1/17.
 */
@Entity
public class StudyContent {
    public @Id long id;
    public String title;
    public String description;
    public String type;
    public boolean isOld;

    public long getId() {
        return id;
    }

    public void setId(long id) {
        this.id = id;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public boolean isOld() {
        return isOld;
    }

    public void setOld(boolean old) {
        isOld = old;
    }
}

File: app/src/main/java/com/ddu/db/entity/StudyContent.kt
package com.ddu.db.entity

import androidx.room.Entity
import androidx.room.PrimaryKey
import org.jetbrains.annotations.NotNull

/**
 * Created by yzbzz on 2018/1/17.
 */
@Entity(tableName = "study_content")
class StudyContent(
        @NotNull
        @PrimaryKey
        var id: Long = 0,
        var title: String? = null,
        var description: String? = null,
        var type: String? = null,
        var isOld: Boolean = false)